### ISSUE TITLE
ModuleNotFoundError: No module named 'git.objects'

### ISSUE THREAD

**Issue by HappyEinara** [gitpython-developers/GitPython#1030]
The release just now might have broken something.

Steps to reproduce:
- Fresh venv, python3.8
- pip install --upgrade gitpython
- python
then:
```
>>> import git
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/vagrant/.virtualenvs/tmpgit/lib/python3.8/site-packages/git/__init__.py", line 41, in <module>
    from git.objects import *               # @NoMove @IgnorePep8
ModuleNotFoundError: No module named 'git.objects'
```

**Comment by michalb-dsg** [comment]
I got the same error

**Comment by Byron** [comment]
Definitely related to the changes coming from #1028 , could @priv-kweihmann have a look?
The tar archive definitely has the 'git/objects' directory, so I am a bit surprised it doesn't get installed.

As immediate action I am rolling the change back.

**Comment by p0tr3c** [comment]
same here :-(

**Comment by Byron** [comment]
v3.1.5 was released with the PR in question reverted. My apologies for the inconvenience!

I hope there is a way to test the usability of releases prior to making them, somehow, it's not the first time a release breaks even though it looks alright. Maybe something like this can be contributed. Personally I do consider setup.py and friends black magic that I cannot touch or tame, and must be relying on contributions to improve the situation. Thanks for your understanding and support.

**Comment by priv-kweihmann** [comment]
> Definitely related to the changes coming from #1028 , could @priv-kweihmann have a look?
> The tar archive definitely has the 'git/objects' directory, so I am a bit surprised it doesn't get installed.
> 
> As immediate action I am rolling the change back.
I will - as this could take some time, it's really better to rollback for now. Will try to include some tests so that this won't happen again

**Comment by Byron** [comment]
I am closing this issue as it was fixed with v3.1.5, which was released about 30 minutes after v3.1.4.
With the PR above we have brought back the reduced package size in v3.1.6, which is likely to be working as we are now testing for git being importable after having been installed (from source).
